### PR TITLE
Remove thiserror usages

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

**Comment by A6GibKm** [comment]
Please add a brief description of why this is desirable to the commit :)

**Comment by bilelmoussaoui** [comment]
> Please add a brief description of why this is desirable to the commit :)

2 less dependencies on all the apps that usually just use anyhow.

File: cairo/src/error.rs [hunk 1]
```diff
@@ -259,20 +269,51 @@ impl From<ffi::cairo_status_t> for Error {
...
 
-#[derive(Error, Debug)]
+#[derive(Debug)]
 pub enum IoError {
-    #[error("Cairo error: {0}")]
-    Cairo(#[from] Error),
-    #[error("IO error: {0}")]
-    Io(#[from] io::Error),
+    Cairo(Error),
+    Io(io::Error),
```
Comment (sdroege) [inline cairo/src/error.rs:275]: You're missing the `Error::source()` / `cause()` implementations. @GuillaumeGomez added those it seems

**Comment by sdroege** [comment]
Let's close this one in favour of https://github.com/gtk-rs/gtk-rs-core/pull/1465, which is more complete.

**Pull request closed without merging**